WebAPI.OData.EFCore/Controllers/OpsTenantController.cs

```diff
@@ -6,9 +6,11 @@
 using Microsoft.AspNetCore.OData.Query;
 using Microsoft.AspNetCore.OData.Routing.Controllers;
 using System.Threading.Tasks;
+using WebAPI.OData.EFCore.Attributes;
 
 namespace WebAPI.OData.EFCore.Controllers
 {
+    [Route("OpsTenant")]
     public class OpsTenantController : ODataController
```

Comment:
Shouldn't have to change except for one new method in each controller and the class level `Route` attribute.

Reply:
I'll look into this

Reply:
Change reverted and created new controllers (`OpsTenantAllowApplyController`, `CoreBuildingAllowApplyController`) for testing the `AllowApplyAttribute`